docs/userguide.md

```diff
@@ -69,12 +69,30 @@ These binaries can be installed via the `download-binaries.sh` script, which
 downloads them to `./bin`:
 
 ```bash
+git clone https://github.com/kubernetes-sigs/federation-v2.git $GOPATH/src/github.com/kubernetes-sigs/federation-v2
+cd $GOPATH/src/github.com/kubernetes-sigs/federation-v2
 ./scripts/download-binaries.sh
 export PATH=$(pwd)/bin:${PATH}
 ```
 
 Or you can install them manually yourself using the guidelines provided below.
 
+`kubefedctl` is the federation command line utility. You can built it from
+the federtion source code or download it from the release site.
+
+If you want to have the latest `kubefedctl`, you can build it by yourself.
+```bash
+make kubefedctl
+```
+Or you can download the latest binary from [release page](https://github.com/kubernetes-sigs/federation-v2/releases).
+```bash
+VERSION=<latest-version>
+wget https://github.com/kubernetes-sigs/federation-v2/releases/download/${VERSION}/kubefedctl.tgz
+tar -zxvf kubefedctl.tgz
+chmod u+x kubefedctl
```

Comment:
Do we need to add a note the the tool is built for Linux only?

Reply:
ok

Reply:
I think we should start releasing a darwin binary with v1beta1.  I've elevated the related issue accordingly:

https://github.com/kubernetes-sigs/federation-v2/issues/373

Reply:
- If we're explaining how to download (vs just pointing at the download page), that suggests explaining how to verify the integrity of the archive with sha256sum/openssl.  It's not clear to me that we need to be so detailed though - it may be sufficient to point someone at the release page and tell them to put it in `~/bin`.

- wget is deprecated in favor of curl on many operating systems, so please use curl instead (likely with -LO args as per the example of our download script).

- Is chmod necessary?  It appears to already have that permission when I extract the tgz.

Reply:
What did you mean for this?
 - add more for `verify the integrity`?
 - remove all details here and just asks them to download and move to `/bin`?

Reply:
I would lean towards the 2nd option (unless you feel strongly for the 1st).

Reply:
I prefer option 1, it follows similar way as `kubectl` installation. And easy for a new user to use.
1. `curl` is used instead of `wget`
2. integrity check is not added because it is not mandatory for using `clusterctl` I think.
3. `chmod +x` is harmless and make sure always get what we want.